### PR TITLE
Sl 1506 manage users

### PR DESCRIPTION
**Pull request by rodrigues-t**
https://zypone.atlassian.net/browse/SL-2105

### CODE REVIEW

File: source/index.html.md [hunk 1]
```diff
@@ -10,13 +10,15 @@ toc_footers:
 
 includes:
   - filters
+  - roles
```
Comment (seyfer) [inline source/index.html.md]: why some are capitalized and some not?

File: source/includes/_roles.md [hunk 2]
```diff
@@ -0,0 +1,158 @@
+# Roles
+
+## Get All Roles
+
+```shell
+curl --request GET \
+  --url 'https://public-api.lumiformapp.com/api/v1/roles' \
```
Comment (seyfer) [inline source/includes/_roles.md:7]: is Role a new endpoint we add recently?
if yes, why we do not add it to v2? 
it was not present in v1, as we develop v2 already, v1 needs to be frozen in the current state and all the additions go to v2.

File: source/index.html.md (hunk 1 above)
Comment (rodrigues-t) [reply source/index.html.md]: will fix

File: source/includes/_roles.md (hunk 2 above)
Comment (rodrigues-t) [reply source/includes/_roles.md:7]: @seyfer No, it is not a new endpoint. For some reason there is no documentation for Role, but it is important for user v2 endpoint. So I included role documentation.

**Review by seyfer — APPROVED** [review]
👍

**Review by tributeee — APPROVED** [review]

**Pull request merged**